src/item.cpp

```diff
@@ -1511,6 +1582,12 @@ nc_color item::color(player *u) const
         ammotype amtype = ammo_type();
         if (u->has_ammo(amtype).size() > 0)
             ret = c_green;
+    } else if(is_item_container()) {    // something we can store items in
+        if(!has_been_opened || is_container_empty()) {
+            ret = c_ltgray;             // unknown contents or we know it's empty
```

Comment:
I want to perhaps make unopened bags a color and just make 'known' bags regular.